data/data/aws/vpc/outputs.tf

```diff
@@ -26,20 +26,12 @@ output "console_sg_id" {
   value = "${aws_security_group.console.id}"
 }
 
-output "aws_lb_private_target_group_arns" {
-  value = "${compact(concat(aws_lb_target_group.api_internal.*.arn, aws_lb_target_group.services.*.arn))}"
+output "aws_lb_target_group_arns" {
+  value = "${compact(concat(aws_lb_target_group.api_internal.*.arn, aws_lb_target_group.services.*.arn, aws_lb_target_group.api_external.*.arn))}"
 }
 
-output "aws_lb_private_target_group_arns_length" {
-  value = "${var.private_master_endpoints ? 2 : 0}"
-}
-
-output "aws_lb_public_target_group_arns" {
-  value = "${compact(concat(aws_lb_target_group.api_external.*.arn))}"
-}
-
-output "aws_lb_public_target_group_arns_length" {
-  value = "${var.public_master_endpoints ? 1 : 0}"
+output "aws_lb_target_group_arns_length" {
+  value = "${var.private_master_endpoints ? 2 : 0} + ${var.public_master_endpoints ? 1 : 0}"
```

Comment:
[e2e-aws][1]:

```
Error: module.masters.aws_lb_target_group_attachment.master: __builtin_StringToInt: strconv.ParseInt: parsing "2 + 1": invalid syntax in:
 ${var.instance_count * var.target_group_arns_length}
```

Looks like I need to tweak the syntax here.

[1]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/609/pull-ci-openshift-installer-master-e2e-aws/1188/build-log.txt

Reply:
> Looks like I need to tweak the syntax here.

Done with 7931855.